cluster/addons/istio/noauth/istio.yaml

```diff
@@ -0,0 +1,2198 @@
+# GENERATED FILE. Use with Kubernetes 1.7+
+# TO UPDATE, modify files in install/kubernetes/templates and run install/updateVersion.sh
+################################
+# Istio system namespace
+################################
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-system
```

Comment:
As we discussed before, for enable this seems fine. But for disable, addon-manager will not do the cleanup when all resources under this new namespace are removed from manifests. Is it acceptable for the alpha phase?